test/numerical/TestGRU.cpp

```diff
@@ -26,6 +26,22 @@ using namespace mlir;
 // Sigmoid
 float sigmoid(float x) { return 1 / (1 + exp(-x)); }
 
+// Build an ONNXConstantOp from an OMTensor.
```

Comment:
Would it make sense to have a Helper.cpp, to be shared among the numerical tests? Maybe it is too much trouble, or just a header file with the inline implementations...

Reply:
Done.